_gsocproposals/2022/proposal_MadAnalysis-mcweights.md

```diff
@@ -1,7 +1,7 @@
 ---
 title: MadAnalysis 5 - Integration of theoretical uncertainty calculation with multiweight integration
 layout: gsoc_proposal
-project: MadAnalysis 5
+project: MadAnalysis5
```

Comment:
This does not work in my test, the MadAnalysis project p[age shows no proposals. This works without this change.

Reply:
That's the opposite on my local site ! I missed something…

Reply:
Well, the best check is the regular project page which renders correctly as it is now: https://hepsoftwarefoundation.org/gsoc/projects/2022/project_MadAnalysis5.html

Reply:
Andrei, your are saying the link following `Corresponding Project` in `proposal_MadAnalysis-mcweights.md` is working without renaming `project_MadAnalysis5.md` to `project_MadAnalysis 5.md` ?

Reply:
No, just the link from the project page to the proposal, which disappears with your patch. But indeed, the back-link from the proposal to the project is not working. So probably you are right to remove the space, only that we should do it also in the project file `project_MadAnalysis5.md` by using `project: MadAnalysis5`